src/module/utils/TwodsixShipActions.ts

```diff
@@ -82,13 +89,35 @@ export class TwodsixShipActions {
 
       /*get characteristic key, default to skill key if none specificed in formula */
       let characteristicKey = "";
+      const charObject = selectedActor?.system["characteristics"] ?? {};
+      //we need an array
+      const charObjectArray = Object.keys(charObject).map(key => {
+        return charObject[key];
+      });
       if(!char) {
         characteristicKey = getKeyByValue(TWODSIX.CHARACTERISTICS, (<Skills>skill.system).characteristic);
       } else {
-        characteristicKey = getCharacteristicFromDisplayLabel(char, selectedActor);
+        //find the most advantageous characteristic to use
+        const charOptions = char.split("|");
+        const candidateCharObjects = [];
+        let candidateCharObject = {};
+
+        for (const charOption of charOptions){
+          candidateCharObject = charObjectArray.find((itm: object) => itm.shortLabel.toUpperCase() === charOption.toUpperCase());
```

Comment:
I don't think we should force cases here and leave that up to user.

Reply:
I just figured there was no reason for case sensitivity here, it will just frustrate the user if the internal string is DEX and they put in dex.  Happy to remove it if we're passionate about it, but I tend to be in the habit of casing string comparisons when case doesn't matter.